packages/docusaurus-theme-classic/src/theme/BlogPostPage/index.js

```diff
@@ -20,7 +20,10 @@ function BlogPostPage(props) {
         <div className="container margin-vert--xl">
           <div className="row">
             <div className="col col--8 col--offset-2">
-              <BlogPostItem frontMatter={frontMatter} metadata={metadata}>
+              <BlogPostItem
+                frontMatter={frontMatter}
+                metadata={metadata}
+                isBlogPostPage>
```

Comment:
Prefer to do `isBlogPostPage={true}`

Reply:
Hmm.. OK, fixed :heavy_check_mark:

Reply:
@yangshun 

CI fails:
```
  26:17  error  Value must be omitted for boolean attributes  react/jsx-boolean-value
```

What have I done wrong?

Reply:
I think our lint rules prefers not explicitly writing the boolean value 🤦‍♂ Sorry let's change it back.

Reply:
Fixed! :+1: